src/Controls/Maps/src/AppHostBuilderExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Maui.Controls.Maps;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.LifecycleEvents;
+using Microsoft.Maui.Maps.Handlers;
+
+#if ANDROID
+using Android.Gms.Common;
+using Android.Gms.Maps;
+#endif
+
+namespace Microsoft.Maui.Controls.Hosting
+{
+	public static partial class AppHostBuilderExtensions
+	{
+		public static MauiAppBuilder UseMauiMaps(this MauiAppBuilder builder)
+		{
+			builder
+				.ConfigureMauiHandlers(handlers =>
+				{
+					handlers.AddMauiMaps();
+				})
+				.ConfigureLifecycleEvents(events =>
+				{
+#if __ANDROID__
+					// Log everything in this one
+					events.AddAndroid(android => android
+						.OnCreate((a, b) =>
+						{
+
+							Microsoft.Maui.Maps.Handlers.MapHandler.Bundle = b;
+#pragma warning disable CS0618 // Type or member is obsolete
+							if (GooglePlayServicesUtil.IsGooglePlayServicesAvailable(a) == ConnectionResult.Success)
```

Comment:
Probably want to get rid of the obsolete method here? I think the replacement is this: GoogleApiAvailability.Instance.IsGooglePlayServicesAvailable(a)

Reply:
Yap good point

Reply:
Fixed